catalog/transformer/catalog-transformer-overlay/src/main/java/ddf/catalog/transformer/OverlayMetacardTransformer.java

```diff
@@ -71,29 +83,106 @@ private BinaryContent overlay(Metacard metacard, Map<String, Serializable> argum
         final BufferedImage image =
                 bufferedImageOptional.orElseThrow(() -> new CatalogTransformerException(
                         "Did not receive an image from the image supplier."));
-        final Builder imageBuilder = Thumbnails.of(image)
-                .scale(1)
-                .imageType(BufferedImage.TYPE_INT_ARGB)
-                .outputFormat(PNG);
-        rotate(imageBuilder, metacard.getLocation());
-        return createBinaryContent(imageBuilder);
+
+        List<Vector> boundary = parseBoundary(metacard.getLocation());
+        BufferedImage tile = createTileFromImageAndBoundary(image, boundary);
+        return createBinaryContent(tile);
     }
 
-    private void rotate(Builder imageBuilder, String wkt) throws CatalogTransformerException {
-        final double angle = GeometryUtils.getRotationAngle(wkt);
-        imageBuilder.rotate(angle);
+    private List<Vector> parseBoundary(String location) throws CatalogTransformerException {
+        final Geometry geometry = parseGeometry(location);
+        if (!canHandleGeometry(geometry)) {
+            throw new CatalogTransformerException("The metacard geometry is not a rectangle.");
+        }
+
+        final Coordinate[] coordinates = geometry.getCoordinates();
+
+        List<Vector> boundary = new ArrayList<>();
+        boundary.add(new BasicVector(new double[] {coordinates[0].x, coordinates[0].y}));
+        boundary.add(new BasicVector(new double[] {coordinates[1].x, coordinates[1].y}));
+        boundary.add(new BasicVector(new double[] {coordinates[2].x, coordinates[2].y}));
+        boundary.add(new BasicVector(new double[] {coordinates[3].x, coordinates[3].y}));
+        return boundary;
     }
 
-    private BinaryContent createBinaryContent(Builder imageBuilder)
-            throws CatalogTransformerException {
-        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    private BufferedImage createTileFromImageAndBoundary(BufferedImage image, List<Vector> boundary) {
+        /*
+         * We transform the image by moving the corners and applying
+         * transparency so that it looks right when laid down as a north-up
+         * rectangular tile (the only thing Cesium currently supports).
```

Comment:
I will remove that part of the comment. Other maps have the ability to put a picture in a boundary you define with a single function call, so I wanted to be reminded that if Cesium adds this functionality or if we switch to something else then we should revisit this.